pyproject.toml

```
@@ -12,9 +12,9 @@ test = [
   "black ~= 22.3",
   "jedi ~= 0.18.0",
   "mypy ~= 0.910",
-  "pylint ~= 2.9.3",
+  "pylint ~= 2.15.10",
   "pytest ~= 7.2.0",
-  "setuptools-rust ~= 0.12.1",
+  "setuptools-rust ~= 1.5.2",
```

I think we tried merging this a while back, but never figured out what the breakage was https://github.com/swift-nav/swift-toolbox/pull/805

We should probably drop this change and merge the rest of this PR, then figure out what's wrong with `setuptools-rust`

```suggestion
  "setuptools-rust ~= 0.12.1",
```

---

im trying to set this up on windows atm 🤔 seems to be missing `console_backend.pdb` but not too certain
will just revert this 👍